tools/protocol-schema-check/res/protocol_schema.toml

```diff
@@ -24,32 +24,32 @@ BandwidthRequestsV1 = 3810915065
 BandwidthSchedulerState = 2982803600
 BandwidthSchedulerStateV1 = 34546280
 BitArray = 3709965115
-Block = 3541579558
-BlockBody = 206872245
-BlockBodyV1 = 2404380656
-BlockBodyV2 = 4061842967
+Block = 1320625425
```

Comment:
all of these are because I removed some type aliases.  The actual structs have not changed.